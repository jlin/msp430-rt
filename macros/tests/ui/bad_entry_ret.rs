#![no_main]

use msp430_rt_macros::entry;

#[entry]
fn main() {}
