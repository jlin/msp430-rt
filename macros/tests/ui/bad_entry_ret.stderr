error: `#[entry]` function must have signature `[unsafe] fn([<ident> : CriticalSection]) -> !`
 --> tests/ui/bad_entry_ret.rs:6:1
  |
6 | fn main() {}
  | ^^^^^^^^^
